app/gateway/postgres/repositories/deliveries_get.go

```diff
@@ -0,0 +1,65 @@
+package repositories
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/techhub-jf/farmacia-back/app/gateway/api/handler/schema"
+)
+
+func (r *DeliveriesRepository) GetAll(ctx context.Context, filters schema.Pagination) ([]*schema.ListDeliveriesResponse, int, error) {
```

Comment:
```suggestion
func (r *DeliveriesRepository) ListAll(ctx context.Context, filters schema.Pagination) ([]schema.ListDeliveriesResponse, int, error) {
```

Não tem necessidade de ser ponteiro